### ISSUE TITLE
Feature Request: MoMMI Liberation

### ISSUE THREAD

**Issue by octareenroon91** [Koriath/-tg-station#4]
Can you make it possible to remove the self-destruct (killswitch) or muteness (keeper) restrictions on non-emagged mommis?

I'm imagining this involving a hacking (wires) system or a multitool pop-up.

Also, in the same way mommi eyes turn red when emagged, can we get rainbow flashing eyes under some condition?

**Comment by Koriath** [comment]
I find that the self destruct on round-start Mommi's to serve no gameplay purpose. Derelict Mommi's, yeah, I can see why.
The muteness, I'm not sure. It's not only part of their charm, it's also a way to further cement that "no involvement" ruleset.

What I am thinking, is someone becoming a master of a specific Mommi. And the Mommi can only talk to him. How and with what, I don't know yet. But befriend a Mommi sounds like a better alternative.

**Comment by octareenroon91** [comment]
Yeah, befriend is close to what I'm thinking too, maybe. The killswitch serves to keep mommi players from ditching or invading the station, given the power to respawn on the derelict.

They're grief control and role enforcement. Maybe a PDA upgrade would be fitting?

**Comment by Koriath** [comment]
PDA upgrade? How so?

By befriend, I mean that their lawset would be sdlightly changed.
Instead of protecting and maintaining a station, they protect and maintaint a carbon.
emagging is clearly the "bad" way of doing it. There could be a "good" way of doing it.

**Comment by Ebinmaymayoverlord** [comment]
What if there's a module made in science that does what you described koriath?

**Comment by octareenroon91** [comment]
In case it was unclear: the killswitch and muteness are controlled by variables. Killswitch=0 and it can go anywhere. Keeper=0 and it can talk with humans, maybe other implications.

A PDA upgrade would mean a mommi 'has' a PDA like the AI 'has' one, or more restricted, like the power to message one human only.

**Comment by Asd396** [comment]
Keeper variable also controls MoMMI anti-griff, preventing them from clicking beings and welderbombing. Keeper also allows other keepers to interact with you. Should be simple enough to make muteness its own variable, I'll take a crack at it tomorrow.
